extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -514,6 +520,18 @@ private static OidcConfigurationMetadata createLocalMetadata(OidcTenantConfig oi
                 oidcConfig.token.issuer.orElse(null));
     }
 
+    private static void fireOidcServerNotAvailableEvent(String authServerUrl) {
+        if (Arc.container().instance(SecurityConfig.class).get().events().enabled()) {
+            if (authServerUrl == null) {
```

Comment:
I'd consider removing this guard since we don't expect it happening, if it will, then we'll handle a bug somewhere up the chain :-)

Reply:
Ok.